integration/test_env_logging.py

```diff
@@ -0,0 +1,810 @@
+import os
```

Comment:
This is a very long file 😅
- The `root_handler` is configured a few times in seemingly the same way. Can this be moved into its own fixture?
- I see that this test uses mocks only with no connection to the DB. As such, can it be moved to the `mock_tests` directory? 
- Can the fixtures and `LogCaptureHandler` be moved into the `conftest.py` file?